cpp/src/arrow/dataset/file_rados_parquet.h

```diff
@@ -128,6 +139,8 @@ class ARROW_DS_EXPORT DirectObjectAccess {
   std::shared_ptr<RadosCluster> cluster_;
 };
 
+/// \brief A ParquetFileFormat implementation that offloads the computation
+/// for the queries onto server Ceph nodes.
```

Comment:
we can say "that offloads the fragment scan operations to the Ceph OSDs"

Reply:
Done.